samples/xrc/myframe.cpp

```diff
@@ -285,6 +286,14 @@ void MyFrame::OnUncenteredToolOrMenuCommand(wxCommandEvent& WXUNUSED(event))
     dlg.ShowModal();
 }
 
+void MyFrame::OnMultipleAccels(wxCommandEvent& WXUNUSED(event))
+{
+    wxString msg;
+    msg.Printf( "You can open this dialog with any of `Ctrl-W` (main), 'Ctrl-Q' or 'Shift-Ctrl-W' (extra) accelerators.");
```

Comment:
Sorry, my fault, I've somehow mistyped the quotes, but they should be the same for all of them, of course:
```suggestion
    msg.Printf( "You can open this dialog with any of 'Ctrl-W' (main), 'Ctrl-Q' or 'Shift-Ctrl-W' (extra) accelerators.");
```